### DISCUSSION TITLE
Fork it

### DISCUSSION THREAD

**Discussion by ZoltanPrime** [bromite/bromite#2637] [General]
There are 18 contributors on this project.  If the main Dev has abandoned it, can someone fork it?  
  
Edit: Someone already forked it.  https://github.com/uazo/cromite

**Comment by Leopold702** [comment]
uazo is not someone, he is one of the two main Bromite developers.

**Reply by Universalizer** [reply]
After Carl as a Project Founder and Main Developer, Many privacy enhancements & collaboration from Uazo.

https://github.com/bromite/bromite/graphs/contributors

csagan5
629 commits    1,836,047 ++    1,751,319 --

uazo
150 commits    44,673 ++    8,369 --

Uazo's active & consistent dedication here https://github.com/uazo/cromite

Carl has neither declare anything, nor archive this Project & Repository. Hobbyist Project in free time.
